DigitalLearningSolutions.Web/ViewModels/Common/TextInputViewModel.cs

```diff
@@ -26,8 +29,8 @@ public TextInputViewModel
             SpellCheck = spellCheck;
             Autocomplete = autocomplete;
             HintText = hintText;
-            ErrorMessage = errorMessage;
-            HasError = hasError;
+            ErrorMessages = errorMessageList;
+            HasError = errorMessageList.Any(message => !string.IsNullOrWhiteSpace(message));

```

Comment:
Just out of curiosity - is this consistent with the ErrorSummary component?

Reply:
Ah, it's not actually.

Reply:
Do you want me to remove the predicate from the Any(), or add it to ErrorSummary?

Reply:
I think it's almost impossible that we'll add a whitespace or null error in the code, so either is fine as long as we make sure that it's consistent across the two components :)